pkg/rules/rego/deprecated-1-22.rego

```diff
@@ -41,11 +46,36 @@ deprecated_api(kind, api_version) = api {
 			"new": "coordination.k8s.io/v1",
 			"since": "1.19",
 		},
-		"LeaseList": {
-			"old": ["coordination.k8s.io/v1beta1"],
-			"new": "coordination.k8s.io/v1",
+		"PriorityClass": {
+			"old": ["scheduling.k8s.io/v1beta1"],
+			"new": "scheduling.k8s.io/v1",
+			"since": "1.14",
+		},
+		"RBAC": {
```

Comment:
This needs to be expanded into 4 actual resources: 
- `ClusterRole`
- `ClusterRoleBinding`
- `Role`
- `RoleBinding`